### DISCUSSION TITLE
Submit back to main awesome list

### DISCUSSION THREAD

**Discussion by sergeyshevch** [wsvincent/awesome-django#140] [Ideas]
I've found that main awesome list (https://github.com/sindresorhus/awesome) doesn't contain any info about this awesome. Maybe you can submit it back to it?

**Comment by jefftriplett** [comment]
Thank you for the idea. I think we have meant to for a while but just hadn't yet. cc @wsvincent want to take this on?

**Comment by wsvincent** [comment]
Yup. On it. Thanks @sergeyshevch for the nudge.

**Reply by jefftriplett** [reply]
@sergeyshevch soon... https://github.com/sindresorhus/awesome/pull/2060

**Comment by wsvincent** [comment]
https://github.com/sindresorhus/awesome/pull/2060#issuecomment-926445905

If someone else wants to make this happen, go for it. I'm 0/2. Not gonna spend further time on the matter.

**Reply by jefftriplett** [reply]
I don't have the time for it. I don't mind feedback, but clearly, our objectives and expectations don't match what the awesome list expects. That's fine, and to each their own, but judging from several recent closed PRs, I'm fine not being associated with it. 🤷

**Reply by sergeyshevch** [reply]
@wsvincent Maybe i can look on it in more details and maybe we can fix some of review comments. I guess i will look on it on next week
